openedx/core/djangoapps/notifications/views.py

```diff
@@ -76,7 +75,7 @@ def get_paginated_response(self, data):
         Return a response given serialized page data with show_preferences flag.
         """
         response = super().get_paginated_response(data)
-        response.data["show_preferences"] = get_show_notifications_tray(self.request.user)
+        response.data["show_preferences"] = ENABLE_NOTIFICATIONS.is_enabled()
```

Comment:
This will not work if `ENABLE_NOTIFICATIONS` flag is enabled for only one course. Use 

`get_show_notifications_tray(self.request.user)`

and in 

```
def get_show_notifications_tray(user):
   # Use ENABLE_NOTIFICATIONS.is_enabled(user) in previous implementation
```